src/shapes/Text/Text.ts

```diff
@@ -79,6 +79,22 @@ const additionalProps = [
   'pathAlign',
 ] as const;
 
+const textLayoutProperties: string[] = [
```

Comment:
why not define it directly on the class?

Reply:
i like the single line of the claration